src/main/resources/assets/duyguji/vanilla.txt

```diff
@@ -80,3 +87,25 @@
 ☪/star-and-crescent
 ☮/peace-symbol
 ☫/farsi-symbol
+⌛/hourglass-done
+⌚/watch
+☄/comet
+✏/pencil
+✎/lower-right-pencil
```

Comment:
I recently started grouping shortcodes by the most intuitive word first, so could you make this `pencil-lower-right`?